cypress/integration/Case1_Sign_up.js

```diff
@@ -13,6 +13,12 @@ describe('Test case 1: Ability to Sign Up', () => {
     SignUpPage.pageHeader.should('contain', 'Sign Up')
     SignUpPage.nameInput.should('be.visible')
     SignUpPage.nextButton.should('be.visible')
+    //Need add wait due issue with next button
```

Comment:
all these waits and clear storage and reload doesnt seem right at all
maybe you need to wait, why do you need to clear storage and reload?
why do you need to wait?

Reply:
This need due to, before input fields are accessible appears spinner, which in cypress is infinite if don't clear cache and reload the page. With just wait, it don't work

Reply:
the spinner has been removed check now

Reply:
@alexpirogovqa  @AnastasiiaOdnoshevna this conversation is still not resolved...
the spinner has been removed
all the "wait" times are not supposed to be necessaray

Reply:
I will remove wait and clear local storage and if all will be good. I will commit changes